types/k6/experimental/websockets.d.ts

```diff
@@ -93,15 +176,15 @@ export enum ReadyState {
  * Type alias describe the types of binary data that can be
  * transmitted over a Websocket connection.
  */
-export type BinaryType = 'arrayBuffer';
+export type BinaryType = 'ArrayBuffer';
```

Comment:
I was confused at first why the right-hand side was a static string, although the left-hand side was a type definition called `SomethingType`. It could be worth renaming it to `BinaryKind` or `SupportedBinaryDataTypes` or something along those lines ❓

Reply:
The WebSocket specifications define the `BinaryType` https://websockets.spec.whatwg.org/#enumdef-binarytype, so I believe it is better to keep that name.

However, it's good to catch :muscle:  since, technically, `BinaryType` isn't a type but an enum (with [the only possible value](https://github.com/grafana/xk6-websockets/blob/main/websockets/websockets.go#L182-L188) as `ArrayBuffer).

Reply:
Then I agree, indeed 🤝 🙇🏻